R/adapter.r

```diff
@@ -232,17 +232,15 @@ construct_s3_adapter <- function() {
   write_function <- function(object, opts) {
     common_s3mpi_package_loader()
 
-    if (is.element('output', names(object))) {
```

Comment:
ls is not recursive and might fail if one if the object in the middle does not exist
and this kind of failture seems to be silent when running syberia (still not sure why)..